### PR TITLE
Added JSConf EU's logo

### PR DESCRIPTION
**Pull request by lukaszklis**

### CODE REVIEW

**Review by usefulthink — APPROVED** [review]
looks good.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_size = 2
+indent_styles = space
```
Comment (Kriesse) [inline .editorconfig:6]: 👍

File: css/_components.jsconfeu.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.c-jsconfeu {
```
Comment (Kriesse) [inline css/_components.jsconfeu.css:1]: If we want to use namespaces, shall we add a link to the README, e.g. this one? http://csswizardry.com/2015/08/bemit-taking-the-bem-naming-convention-a-step-further/#namespaces
So we make our class structure transparent.

Comment (Kriesse) [inline css/_components.jsconfeu.css:1]: If we want to use namespaces, shall we add a link to the README, e.g. this one? http://csswizardry.com/2015/08/bemit-taking-the-bem-naming-convention-a-step-further/#namespaces
So we make our class structure transparent.

**Review by Kriesse — APPROVED** [review]
LGTM

**Pull request merged**